### DISCUSSION TITLE
Is it possible to skip preloading but not memoization?

### DISCUSSION THREAD

**Discussion by dgmora** [jnunemaker/flipper#729] [Q&A]
I would like to skip preloading in most cases, but not memoization. Folowing the guides, I did:

```
  config.flipper.memoize = ->(request) {
    request.path =~ %r{/some_path$}  ||
  }
```
This skips preloading for all requests except `some_path`, but it also skips memoization. Is it possible to skip preloading for everything except for `some_path`, but keep memoization?

**Comment by jnunemaker** [comment]
@bkeepers these are tied together. Should we do the same thing for config.flipper.preload as memoize?

**Reply by dgmora** [reply]
To give more context, somewhere I iterate a list and some levels 'down' I check `Flipper.enabled?(:some_feature, user)`. So I end up with a few too many redis calls. I don't want to preload everything, but I would like to memoize it.

**Reply by jnunemaker** [reply]
Yep, that makes total sense to me. I'll try to get a PR together. Sorry for slow reply here.

**Comment by jnunemaker** [comment]
I've got a PR for this https://github.com/flippercloud/flipper/pull/771. Once that is approved you should be able to do what you want.
